demo/rag/embedding_function_factory.py

```
@@ -38,77 +44,67 @@ def _ensure_model_loaded(self):
-            client = OpenAI(
-                api_key=self.api_key,
-                base_url=self.base_url
-            )
-            response = client.embeddings.create(
-                model=self.model_name,
-                input=["test"]
-            )
+            client = OpenAI(api_key=self.api_key, base_url=self.base_url)
+            response = client.embeddings.create(model=self.model_name, input=["test"])
             self._dimension = len(response.data[0].embedding)
         except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)